src/tools/opt-dist/src/metrics.rs

```diff
@@ -0,0 +1,131 @@
+use crate::timer::TimerSection;
+use camino::Utf8Path;
+use std::time::Duration;
+
+// Sure would be nice to share these data structures with bootstrap!
```

Comment:
put it in `src/tools/build_helper` maybe to share?

Reply:
That's a good idea. @jyn514 Can I do that? :)